scripts/release-build.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+const directory = './build/';
+
+const verbatimFiles = ['LICENSE', 'README.md', 'index.d.ts', 'types.d.ts'];
+
+// Returns a string representing data ready for writing to JSON file
+function createDataBundle() {
+  const bcd = require('../index.js');
+  const string = JSON.stringify(bcd);
+  return string;
+}
+
+// Returns a promise for writing the data to JSON file
+async function writeData() {
+  const dest = path.resolve(directory, 'data.json');
+  const data = createDataBundle();
+  await fs.writeFile(dest, data);
+}
+
+// Returns an array of promises for copying of all files that don't need transformation
+async function copyFiles() {
+  for (const file of verbatimFiles) {
+    const src = path.join('./', file);
+    const dest = path.join(directory, file);
+    await fs.copyFile(src, dest);
+  }
+}
+
+function createManifest() {
+  const full = require('../package.json');
+  const minimal = { main: 'data.json' };
+
+  const minimalKeys = [
+    'name',
+    'version',
+    'description',
+    'repository',
+    'keywords',
+    'author',
+    'license',
+    'bugs',
+    'homepage',
+    'types',
+  ];
+
+  for (const key of minimalKeys) {
+    if (key in full) {
+      minimal[key] = full[key];
+    } else {
+      throw `Could not create a complete manifest! ${key} is missing!`;
+    }
+  }
+  return JSON.stringify(minimal);
+}
+
+async function writeManifest() {
+  const dest = path.resolve(directory, 'package.json');
+  const manifest = createManifest();
+  await fs.writeFile(dest, manifest);
+}
+
+async function main() {
+  // Remove existing files, if there are any
+  await fs
+    .rmdir(directory, {
+      force: true,
+      recursive: true,
```

Comment:
By switching Node versions (with `nodenv`), I discovered that the `recursive` option doesn't work on Node 10. But it's EOL in ~2 weeks. I think we can tolerate the (unlikely) annoyance for that long.

Reply:
Actually, now that I think on it some more: I guess it doesn't matter at all. If all we publish is a `.json` file, then we can adopt whatever version of Node we want for development.

(Though this also suggests that, should we desire to expose utilities for working with compat data, it would need to be as a separate package.)

Reply:
> By switching Node versions (with nodenv), I discovered that the recursive option doesn't work on Node 10.

Yes, the [docs](https://nodejs.org/api/fs.html#fs_fspromises_rmdir_path_options) say `recursive` was introduced in 12.

> If all we publish is a .json file, then we can adopt whatever version of Node we want for development.

I'll confess: it was my secret plan all along :) . More recent NodeJS versions have nice features, so I wanted to simplify transition to modern NodeJS should you ever choose to.

> (Though this also suggests that, should we desire to expose utilities for working with compat data, it would need to be as a separate package.)

Does BCD expose any utilities now?

Reply:
> I'll confess: it was my secret plan all along :)

😄 Yeah, there's a ton of hidden benefits to this. For example, we could add internal-only fields to the schema that might make working with the data more ergonomic, but filter it out before building the final data for consumers.

> Does BCD expose any utilities now?

No, not yet. It's something we've maybe inched toward (see https://github.com/mdn/browser-compat-data/pull/9441), but have been sort of cautious about it. I rather like the idea of decoupling the data from such utilities anyway and this PR would sort of force that decision.